main.cpp

```diff
@@ -4,15 +4,15 @@
 #include "QSPI.h"
 
 // define what flash we got
-#define N25Q128A
+#define MX25R6435F
```

Comment:
can we keep here N25 as it was and this added also? I would like to have all in one for now - this will serve as a good example for further device driver work and tests that needs to be done